service/src/main/kotlin/fi/espoo/evaka/invoicing/service/IncomeQueries.kt

```diff
@@ -62,12 +62,25 @@ WITH latest_income AS (
     FROM placement pl
     JOIN service_need sn ON pl.id = sn.placement_id AND daterange(sn.start_date, sn.end_date, '[]') @> :dayAfterExpiration
     JOIN service_need_option sno ON sn.option_id = sno.id AND sno.fee_coefficient > 0
-    JOIN guardian g ON g.child_id = pl.child_id
-    JOIN latest_income i ON i.person_id = g.guardian_id
+    
+    -- head of child
+    LEFT JOIN fridge_child fc_head ON (
```

Comment:
This should be a `JOIN`, not `LEFT JOIN`. If the child doesn't have a head of child, then the query shouldn't return anything.